spec/model/answer/money_spec.rb

```diff
@@ -49,30 +49,63 @@
         expect(instance.humanize).to eql("£1,523.42")
       end
     end
+
     context "no pence" do
       let(:money_input) { '1523.00' }
       it "rounds down amounts of money correctly" do
         expect(instance.humanize).to eql("£1,523")
       end
     end
+
+    context "for value with a trailing dot" do
+      let(:money_input) { '1523.' }
+      it "ignores the trailing dot" do
+        expect(instance.humanize).to eql("£1,523")
+      end
+    end
   end
 
-  describe "errors" do
-    context "invalid formatting" do
+  describe "parsing" do
+    context "no digits" do
       let(:money_input) {"Loads'a'money"}
 
-      it "Has errors" do
+      it "raises an error" do
         expect(instance.error).to eql("Invalid format")
       end
     end
 
-    context "no input" do
+    context "empty input" do
       let(:money_input) { nil }
 
-      it "Has errors" do
+      it "raises an error" do
         expect(instance.error).to eql("Please answer this question")
       end
     end
+
+    context "float with a trailing dot" do
+      let(:money_input) { "200." }
+
+      it "raises no errors" do
+        expect(instance.error).to eql(nil)
```

Comment:
Is there any reason not to explicitly expect the value, rather than the absence of errors?

We should define that 200. is a valid input, but also be clear about what that will be parsed as.

Eg,